keyboards/keychron/c3_pro/c3_pro.c

```diff
@@ -110,3 +110,29 @@ bool led_matrix_indicators_advanced_kb(uint8_t led_min, uint8_t led_max) {
 }
 
 #endif // LED_MATRIX_ENABLE && CAPS_LOCK_LED_INDEX
+
+void keyboard_post_init_kb(void) {
+    setPinOutputPushPull(LED_MAC_OS_PIN);
+    setPinOutputPushPull(LED_WIN_OS_PIN);
+    writePin(LED_MAC_OS_PIN, !LED_OS_PIN_ON_STATE);
+    writePin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
+    keyboard_post_init_user();
+}
+
+void housekeeping_task_kb(void) {
+    if (default_layer_state == (1U << 0)) {
+        writePin(LED_MAC_OS_PIN, LED_OS_PIN_ON_STATE);
+        writePin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
+    }
+    if (default_layer_state == (1U << 2)) {
```

Comment:
Maybe use the enum to make it clear where the 2 comes from?

Suggested change
```suggestion
    if (default_layer_state == (1U << WIN_BASE)) {
```